sklearn/metrics/cluster/tests/test_supervised.py

```diff
@@ -370,3 +379,70 @@ def test_check_clustering_error():
 
     with pytest.warns(UserWarning, match=msg):
         check_clusterings(wavelength, noise)
+
+
+def test_pair_confusion_matrix():
+    n = 10
+    N = n * n
```

Comment:
I made this
```
n = 10
N = n ** 2
```
because this makes the dependency of n and N clear. I prefer this to n,N = 10,100 because that might leave the impression that n and N can be set independently.